mingw-w64-git/PKGBUILD

```diff
@@ -97,17 +99,23 @@ i686)
 x86_64)
   SHAREDIR=/mingw64/share/git
   ;;
+aarch64)
+  SHAREDIR=/clangarm64/share/git
+  ;;
 esac
 
 prepare () {
 	cd "$srcdir/git" &&
 
 	cat >config.mak <<-EOF
-	USE_ASCIIDOCTOR = YesPlease
+	USE_ASCIIDOCTOR = $([[ "$MSYSTEM" == "CLANGARM64" ]] || echo "YesPlease")
```

Comment:
It would make a lot of sense, I think, to start with a preparatory commit that replaces `USE_ASCIIDOCTOR = YesPlease` with `USE_ASCIIDOCTOR = $USE_ASCIIDOCTOR` and then defines `USE_ASCIIDOCTOR=YesPlease`  somewhere at the top of the file.

Then, the final commit can describe in a succinct and elegant manner what the differences between clang and non-clang are, by having a single clang-specific conditional block at the top of the file (and an `else` block for the non-clang case).